.github/workflows/ci.yml

```
@@ -0,0 +1,41 @@
+name: CI
+on: [push]
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14.17.x
+      - name: Yarn install
+        uses: borales/actions-yarn@v4
+        with:
+          cmd: install
+      - run: yarn check
+      - run: yarn lint
+      - run: yarn flow
+      - run: yarn compile
+      - run: yarn test
+      - run: yarn dist
```

normally I'd just use a single run action to do that, but I don't see any issue with the way you've done it.
You might see a few other files that look like this:
```suggestion
      - run: |
          yarn check
          yarn lint
          yarn flow
          yarn compile
          yarn test
          yarn dist
```
Just thought I'd show the different way